### ISSUE TITLE
Error: Deploy Failedhttps://github.com/AzureCosmosDB/scenario-based-labs/issues

### ISSUE THREAD

**Issue by markjbrown** [AzureCosmosDB/scenario-based-labs#17]
Got this error when running the deploy.ps1. Not sure if this is related to the typo in the subscription name. 

Deploying [labdeploy5.json] - Please wait
Deploying [labdeploy5.json] finished with status Succeeded
Deploying [labdeploy.json] - Please wait
az : ERROR: Deployment failed. Correlation ID: 1b7a4aa0-4fec-4ef4-8458-ab09ba01b74a. {
At C:\scenario-based-labs-master\Retail\deploy\deploy.ps1:173 char:25
+ ... $result = $(az group deployment create --name $deployId --resource-gr ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (ERROR: Deployme...ab09ba01b74a. {:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
  "error": {
    "code": "ResourceNotFound",
    "message": "The Resource 'Microsoft.Web/sites/s2webjmzhc66ozampy' under resource group 'mjb_s2_retail' was not found."
  }
}
Deploying [labdeploy.json] finished with status Failed
Deployment failed: Please enter the suffix that was created for the resource group:

**Comment by markjbrown** [comment]
This error is reproducible even with the correct Subscription Name in the parameters.

**Comment by markjbrown** [comment]
Update: The functions app this script is looking for is actually provisioned. I do not know why the script cannot find it however.

**Comment by givenscj** [comment]
Let's run through this together so I can see what is happening in your specific subscription.  This is an ARM template error outside the powershell script.

**Comment by markjbrown** [comment]
ok, let me try again Wednesday in this sub and see if I can repro. Thanks!

**Comment by givenscj** [comment]
Turns out you are not crazy.  We attempted this in a new spektra environment and the ARM template fails like yours.  There seems to be some kind of bug in ARM templates with system identities, we are researching how to get around the issue.

**Comment by givenscj** [comment]
This has been resolved in the new ARM templates and script.
